packages/server/src/stages/config/index.ts

```diff
@@ -43,14 +57,26 @@ export const createStageConfig: Stage = ({config, processNewFile, processNewChil
         contents: Buffer.from(`
 const {withBlitz} = require('@blitzjs/server');
 const config = require('./blitz.config.js');
-module.exports = withBlitz(config);
+module.exports = withBlitz(config.default ? config.default : config);
         `),
       }),
     )
   }
 
   // No need to filter yet
-  const stream = transform.file((file) => {
+  const stream = transform.file(async (file) => {
+    if (isBlitzTsConfigPath(file.path)) {
+      const res = await bundle(file.path, {
+        quiet: true,
+        externals: ["@blitzjs/server", "@next/bundle-analyzer"], // FIXME: Figure out why this wouldn't work? Do we need to bundle everything?
```

Comment:
Not sure what to do here. `ncc` failed to bundle these 3rd party modules, although it shouldn't have to because they're already built. `ncc` doesn't seem to provide a nice way to do this (https://github.com/vercel/ncc/issues/409) so I'm considering:
* Maintain this list manually, thinking that there shouldn't be a lot of 3rd party modules that the config will be expected to import (mostly just auth stuff?)
* Programmatically get all installed modules with `yarn list --json` or similar

What are your thoughts? @flybayer

Reply:
Maybe we should do this first - outside the file-pipeline?